pkg/kubelet/dockershim/libdocker/client.go

```diff
@@ -71,11 +71,15 @@ type Interface interface {
 // Get a *dockerapi.Client, either using the endpoint passed in, or using
 // DOCKER_HOST, DOCKER_TLS_VERIFY, and DOCKER_CERT path per their spec
 func getDockerClient(dockerEndpoint string) (*dockerapi.Client, error) {
+	clientOpts := []dockerapi.Opt{
+		dockerapi.FromEnv,
+		dockerapi.WithAPIVersionNegotiation(),
```

Comment:
it doesn't look like the previous invocation of NewClientWithOpts enabled version negotiation... what behavior does that change

Reply:
same as above I'll drop the improvements and just keep this PR simpler :)

Reply:
we can add in version negotiation stuff if needed later. basically it enables the client to "adjust" to a docker api server that is of lower version instead of failing hard.